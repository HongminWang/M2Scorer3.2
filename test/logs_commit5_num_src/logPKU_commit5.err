 Time: 0hours 0mins 17.049654007seconds 
