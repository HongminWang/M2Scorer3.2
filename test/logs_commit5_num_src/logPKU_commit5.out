num_source =  30144
Precision   : 0.3221
Recall      : 0.1365
F_0.5       : 0.2532
